nav_msgs/OccupancyGrid raw_map
int32 minclasssize
---
geometry_msgs/Point[] frontiers
